packages/components/alert-dialog/src/AlertDialog.doc.mdx

```diff
@@ -90,6 +90,36 @@ import { AlertDialog } from '@spark-ui/alert-dialog'
 
 <Canvas of={stories.Usage} />
 
+## Uncontrolled
+
+Use `defaultOpen` prop to set the default state of the dialog. Optionally `onOpenChange` prop can be used to know when the state of the dialog changes.
+
+<Canvas of={stories.Uncontrolled} />
+
+## Controlled
+
+Use `open` and `onOpenChange` props to control the state of the dialog.
+
+<Canvas of={stories.Controlled} />
+
+## Escape
+
+Use `onEscapeKeyDown` prop to listen to when the escape key is pressed. Optionally, the dialog could be prevented from closing by calling `event.preventDefault`.
+
+<Canvas of={stories.Escape} />
+
+## Open focus
+
+Use `onOpenAutoFocus` prop to listen to when focus moves to the less destructive action after opening. Optionally, the autofocus behavior could be prevented by calling `event.preventDefault`.
```

Comment:
Not sure if we should explain this "Optionally, the autofocus behavior could be prevented by calling `event.preventDefault`" @Powerplex WDYT ?

Reply:
I think we discussed this in the meantime, but we new way to order stories this example could belong to the "Advanced usage" section to make it less "important"